src/type.h

```diff
@@ -158,4 +158,117 @@ class TypeSChar : public IntegralTypeHelper<signed char> {
     void dbgDump();
 };
 
+class TypeUChar : public IntegralTypeHelper<unsigned char> {
+  public:
+    TypeUChar(bool _is_static, CVQualifier _cv_qual)
+        : IntegralTypeHelper(_is_static, _cv_qual) {}
+
+    IntTypeID getIntTypeId() final { return IntTypeID::UCHAR; }
+    std::string getName() final { return "unsigned char"; }
```

Comment:
TODO for future changes: decouple language representation of the type from back-end type. I.e. for different languages the name ofthe type and the suffix might be different.

Reply:
Added a TODO note in the appropriate commit.